src/regions/ca/info/ca-on/clinics.md

```diff
@@ -4,4 +4,6 @@ Ontario will open six dedicated COVID-19 assessment centres in the coming days a
 
 The first wave of assessment centres will be located at William Osler Health System’s Peel Memorial site, The Ottawa Hospital, North York General Hospital, Mackenzie Health in Richmond Hill and within the Scarborough Health Network and Trillium Health Partners network in Mississauga.
 
+According to this [source](https://www.blogto.com/city/2020/03/map-coronavirus-test-centres-toronto/), Ontario will launch a COVID-19 online self-assessment tool in the next few days.
```

Comment:
@hollyorlando this is it https://www.ontario.ca/page/2019-novel-coronavirus-covid-19-self-assessment